react-common/components/share/ShareInfo.tsx

```diff
@@ -109,6 +109,19 @@ export const ShareInfo = (props: ShareInfoProps) => {
         }
     };
 
+    const handleMultiplayerShareClick = async () => {
+        // TODO: This won't work on staging (parseScriptId domains check doesn't include staging urls)
```

Comment:
```suggestion
        // TODO multiplayer: This won't work on staging (parseScriptId domains check doesn't include staging urls)
```

nit: just makes it easier to search for these later.